zerver/lib/bugdown/api_code_examples.py

```diff
@@ -10,6 +12,11 @@
 import zerver.openapi.python_examples
 from zerver.lib.openapi import get_openapi_fixture, openapi_spec
 
+if settings.RUNNING_OPENAPI_CURL_TEST:  # nocoverage
+    from zerver.openapi.curl_param_value_generators import patch_openapi_params
+
+from django.conf import settings
```

Comment:
We don't need to import settings twice.  Also, if we're doing the import in a conditional, it's probably cleaner to just do it once below.